clang/test/Index/annotate-comments-objc.m

```diff
@@ -56,7 +56,6 @@ void functionFromMacro(void) { \
       typedef struct Struct_notdoxy Struct_notdoxy; \
     }
 
-/// IS_DOXYGEN_NOT_ATTACHED
```

Comment:
I think it would be worth adding another test case where we have this one and check that it uses this comment over the one in the macro.